### PR TITLE
fix: Setting activeRowId to be reactive

### PR DESCRIPTION
**Pull request by bdow**

### CODE REVIEW

File: src/components/Table/GridTable.test.tsx [hunk 1]
```diff
@@ -1592,6 +1593,29 @@ describe("GridTable", () => {
     // But the 2nd added row did not
     expect(row(r, 2).getAttribute("data-render")).toEqual("1");
   });
+
+  it("reacts to setting activeRowId", async () => {
+    const activeRowIdRowStyles: GridRowStyles<Row> = {
+      data: {
+        onClick: (row, api) => {
+          api.setActiveRowId(`${row.kind}`);
```
Comment (stephenh) [inline src/components/Table/GridTable.test.tsx:1601]: Kinda expect to see row.id in here?

Comment (bdow) [reply src/components/Table/GridTable.test.tsx:1601]: oh yea.. i wonder why this worked....

Comment (bdow) [reply src/components/Table/GridTable.test.tsx:1601]: Ahh, because the second color is the same as the first... whoops!

Comment (bdow) [reply src/components/Table/GridTable.test.tsx]: Okay, all fixed.

**Review by stephenh — APPROVED** [review]

**Pull request merged**